manifests/certonly.pp

```diff
@@ -14,12 +14,16 @@
 # [*additional_args*]
 #   An array of additional command line arguments to pass to the
 #   `letsencrypt-auto` command.
+# [*manage_cron*]
+#   Boolean indicating whether or not to schedule cron job for renewal, 
+#   runs daily but only renews if near expiration e.g within 10 days. 
 #
 define letsencrypt::certonly (
   Array[String]                           $domains          = [$title],
   Enum['apache', 'standalone', 'webroot'] $plugin           = 'standalone',
   String                                  $letsencrypt_path = $letsencrypt::path,
   Optional[Array[String]]                 $additional_args  = undef,
+  Boolean                                 $manage_cron      = $letsencrypt::manage_cron,
```

Comment:
This seems to be a per-cert setting, let's move the default value into this class and remove the `$manage_cron` parameter from `init.pp` and it's default from `params.pp`.